ember-flatpickr/src/components/ember-flatpickr.ts

```diff
@@ -92,27 +92,30 @@ export default class EmberFlatpickr extends Component<EmberFlatpickrArgs> {
     // Require that users pass a date
     assert(
       '<EmberFlatpickr> requires a `date` to be passed as the value for flatpickr.',
-      date !== undefined
+      date !== undefined,
     );
 
     // Require that users pass an onChange
     assert(
       '<EmberFlatpickr> requires an `onChange` action or null for no action.',
-      onChange !== undefined
+      onChange !== undefined,
     );
 
     // Wrap is not supported
     assert(
       '<EmberFlatpickr> does not support the wrap option. Please see documentation for an alternative.',
-      wrap !== true
+      wrap !== true,
     );
 
     // Pass all values and setup flatpickr
     scheduleOnce('afterRender', this, this._setFlatpickrOptions, element);
   }
 
-  _setFlatpickrOptions(element: HTMLInputElement): void {
-    const fastboot = getOwner(this)?.lookup('service:fastboot') as unknown as (FastbootService | undefined);
+  @waitFor
```

Comment:
@AmauryD can you please explain this `waitFor` here? I thought test waiters would be in test files or test helpers?

Reply:
Due to the change of _setFlatpickrOptions to async, it caused issues with one test. I needed to add a waitFor statement in the suite. However, for user convenience, I preferred to add @waitFor from @ember/test-waiters to the addon code and do not change the test suite, as there's a chance that consumers' test suites might also break.

What do you think ?

Reply:
@AmauryD I don't think application code should be handling cases for testing. If anything, we should add some of the waiting to the test helpers we ship, but I don't think we want to add `waitFor` to the addon code. I could be wrong, but that addon sounds like it is for test helpers.

Reply:
I'm fine with it. 
Adding the waiting logic to the current test-support would be great. But i currently don't know how i would check that the operation of the flatpickr component is done without service level code or some trickery.